### PR TITLE
Prepare release of v2.0.0

### LINKED ISSUE #12: Release CorporaCoCo 2.0
Prepare for release:

* [x] Finish documentation / vignettes
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_CorporaCoCo.html)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)
* [x] [`urlchecker::url_check()`](https://github.com/r-lib/urlchecker)
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`
* [x] `revdepcheck::revdep_check(num_workers = 4)`
* [x] Update `cran-comments.md`
* [x] Merge #13 #17 

Submit to CRAN:

* [x] ~~`usethis::use_version('major')`~~ (changed manually prior to this issue)
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [x] Add CRAN-SUBMISSION file to master
* [x] Create 2.0 release/tag in GitHub
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_news_md()`
* [ ] `usethis::use_dev_version()`
* [ ] Finish blog post
* [ ] Tweet
* [ ] Add link to blog post in pkgdown news menu

### PR DESCRIPTION
**Pull request by mike-allaway**
Any changes needed to be made as part of submission of v2.0.0 to CRAN. See #12 for more info.

### CODE REVIEW

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ name: R-CMD-check
 
 jobs:
   R-CMD-check:
-    runs-on: macOS-latest
+    runs-on: macOS-10.15
```
Comment (edmondac) [inline .github/workflows/R-CMD-check.yaml:15]:
```suggestion
    runs-on: ubuntu-18.04
```

File: .github/workflows/R-CMD-check.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ name: R-CMD-check
 
 jobs:
   R-CMD-check:
-    runs-on: macOS-latest
+    runs-on: ubuntu-18.04
```
Comment (edmondac) [inline .github/workflows/R-CMD-check.yaml]:
```suggestion
    runs-on: macOS-10.15
```

File: DESCRIPTION [hunk 3]
```diff
@@ -35,6 +36,5 @@ Suggests:
 VignetteBuilder:
     R.rsp,
```
Comment (edmondac) [inline DESCRIPTION]:
```suggestion
```

**Comment by mike-allaway** [comment]
We ran `rhub::check_for_cran()` successfully, so now can merge

File: DESCRIPTION [hunk 4]
```diff
@@ -3,18 +3,19 @@ Package: CorporaCoCo
 Type: Package
 Title: Corpora Co-Occurrence Comparison
 Version: 2.0
-Date: 2021-04-29
+Date: 2021-09-08
```
Comment (mike-allaway) [inline DESCRIPTION]:
```suggestion
Date: 2021-09-21
```

**Pull request merged**